src/main/java/javaserver/ResponseBuilders/DirectoryResponder.java

```diff
@@ -0,0 +1,49 @@
+package javaserver.ResponseBuilders;
```

Comment:
Now that you've renamed these objects as `Responders`, it might make sense to rename your package for consistency and because you also have another object named `ResponseBuilder`.